documentation/docs/13-configuration.md

```diff
@@ -224,6 +224,7 @@ See [Prerendering](/docs/page-options#prerender). An object containing zero or m
 
 - `concurrency` — how many pages can be prerendered simultaneously. JS is single-threaded, but in cases where prerendering performance is network-bound (for example loading content from a remote CMS) this can speed things up by processing other tasks while waiting on the network response
 - `crawl` — determines whether SvelteKit should find pages to prerender by following links from the seed page(s)
+- `default` — set to `true` to prerender every page without `export const prerender = false`
```

Comment:
I wonder if we should combine `default` and `enabled` into a single option. E.g. with values like `true | false | 'never'` or `'default-true' | 'default-false' | 'never'`

Reply:
I think I prefer the booleans, personally